index.html

```diff
@@ -184,6 +187,42 @@ <h3>
           </tr>
         </tbody>
       </table>
+      <h4>
+        [=Relevant global object=]'s [=navigable=]'s [=top-level traversable=]
+      </h4>
+      <p>
+        The following internal slots are added to the [=relevant global
+        object=]'s [=navigable=]'s [=top-level traversable=].
+      </p>
+      <table class="simple">
+        <thead>
+          <tr>
+            <th>
+              Internal slot
+            </th>
+            <th>
+              Description
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <dfn data-dfn-for="Document">[[\PostureOverride]]</dfn>
+            </td>
+            <td>
+              Posture to override posture defined by hardware. Used in
+              automation. Possible values:
+              <li>"{{DevicePostureType/continuous}}"
```

Comment:
This markup is incomplete, you need a parent list element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li

Reply:
Added in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/3f51f595e803619a4190efbcc400f2ff0ba66306